### ISSUE TITLE
ADB parsing error when ADB is not initialized.

### ISSUE THREAD

**Issue by pedrovgs** [pedrovgs/AndroidWiFiADB#3]
Here there is a log trace which reflects this issue:

```
6:45:19 Android WiFi ADB: There are no devices connected with a USB cable.
오후 6:45:21 Android WiFi ADB: Unable to connect device 'mon not running. starting it now on port 5037 *'. Review your WiFi connection.
오후 6:45:21 Android WiFi ADB: Unable to connect device 'mon started successfully *'. Review your WiFi connection.
오후 6:45:21 Android WiFi ADB: There are no devices connected with a USB cable.
USB cable.
오후 6:45:45 StringIndexOutOfBoundsException: String index out of range: -5﻿

```

**Comment by vrunoa** [comment]
What if you use `adb start-server` before calling the other commands ?

**Comment by pedrovgs** [comment]
Fixed in the Release 1.1 I've just published :)
